app/assets/stylesheets/helpers/_header.scss

```diff
@@ -88,10 +88,13 @@
       display: block;
       margin: 0;
       border: 0;
-      height: 35px;
+      min-height: 36px;
       padding: 6px 0;
-      @include ie-lte(7){
-        height: 23px;
+      @include ie-lte(8) {
```

Comment:
Previously IE8 was getting a 35px `height`, but now we’re setting a 24px `min-height`. Does its handling of `height` and `min-height` differ on inputs?